libvast/test/data.cpp

```diff
@@ -404,3 +404,43 @@ TEST(pack / unpack) {
   REQUIRE_EQUAL(unpack(*flatbuffer, x2), caf::none);
   CHECK_EQUAL(x, x2);
 }
+
+TEST(get_if) {
+  // clang-format off
+  auto x = record{
+    {"foo", "bar"},
+    {"baz", record{
+      {"qux", int64_t{42}},
+      {"quux", record{
+        {"quuux", 3.14}
```

Comment:
NB: If you add a trailing comma here it should format the same way even with clang-format enabled for the section.

Reply:
I don't see a way to get the same formatting. clang-format always puts `record` on a separate line.

Reply:
Right, I see what you mean. The result is then this:

```cpp
auto x = record{
  {"foo", "bar"},
  {"baz",
   record{
     {"qux", int64_t{42}},
     {"quux",
      record{
        {"quuux", 3.14},
      }},
   }},
};
```

I don't have a strong enough opinion on this to force a personal preference to avoid disabling clang-format, so feel free to just resolve.